awx/ui_next/src/screens/Job/Job.jsx

```diff
@@ -78,20 +83,10 @@ function Job({ i18n, lookup, setBreadcrumb }) {
     );
   }
 
-  if (lookup && job) {
```

Comment:
Since `lookup` was always `undefined`, our code never entered this block so I removed this as well. `JobTypeRedirect` component located in the parent seemed to handle this exact redirect logic.